src/Image/Image.php

```diff
@@ -111,7 +111,8 @@ public function height(): int
 	 */
 	public function html(array $attr = []): string
 	{
-		return Html::img($this->url(), $attr);
+		$url = $this->url() ?? '';
+		return Html::img($url, $attr);
```

Comment:
`Html::img` only accepts strings

Reply:
Does this method make sense when no URL is defined? Shouldn't it rather also return `null` in this case? Would be a breaking change, but I think it makes sense.

Reply:
Yea was wondering the same but I was afraid it might be too confusing and hard to track down for developers if the whole HTML output disappears/doesn't show up. Clearer to understand that something with the URL must be wrong if the HTML renders with empty `src` attribute.

Reply:
Or maybe we should throw an exception that points to the issue?

Reply:
I'm fine with throwing an exception instead. I guess that wouldn't be a breaking change, would it? Cause it should throw a PHP error anyways already, right?

Reply:
I agree. We already have the `string` type hint in Kirby 3.7.